gordon/interfaces.py

```diff
@@ -0,0 +1,86 @@
+from zope.interface import Attribute
+from zope.interface import Interface
+
+
+class IEventMessage(Interface):
```

Comment:
Could we get a docstring for the class?

Reply:
Good idea! I added a docstring, I went for communicating intent and descriptiveness, do you think it explains the role of EventMessages?